openmetadata-ui/src/main/resources/ui/src/pages/TagsPage/TagsPage.tsx

```diff
@@ -118,11 +118,15 @@ const TagsPage = () => {
 
   const fetchCurrentClassificationPermission = async () => {
     try {
-      const response = await getEntityPermission(
-        ResourceEntity.CLASSIFICATION,
-        currentClassification?.id as string
-      );
-      setClassificationPermissions(response);
+      if (currentClassification?.id) {
+        const response = await getEntityPermission(
+          ResourceEntity.CLASSIFICATION,
+          currentClassification.id as string
+        );
+        setClassificationPermissions(response);
+      } else {
+        return;
+      }
```

Comment:
Here we can add a condition for early return.
```suggestion
      const fetchCurrentClassificationPermission = async () => {
      if(!currentClassification?.id) return;
    try {
      const response = await getEntityPermission(
        ResourceEntity.CLASSIFICATION,
        currentClassification?.id as string
      );
      setClassificationPermissions(response);
    } catch (error) {
      showErrorToast(error as AxiosError);
    }
  };
```

Reply:
Sure